.evergreen/install-dependencies.sh

```diff
@@ -9,9 +9,12 @@ export NODE_LTS_VERSION=${NODE_LTS_VERSION:-16}
 # npm version can be defined in the environment for cases where we need to install
 # a version lower than latest to support EOL Node versions.
 
-# If NODE_LTS_VERSION is numeric and less than 18, default to 9. Do not override if it is already set.
+# If NODE_LTS_VERSION is numeric and less than 18, default to 9, if less than 20, default to 10.
+# Do not override if it is already set.
 if [[ "$NODE_LTS_VERSION" =~ ^[0-9]+$ && "$NODE_LTS_VERSION" -lt 18 ]]; then
     export NPM_VERSION=${NPM_VERSION:-9}
+elif [[ "$NODE_LTS_VERSION" =~ ^[0-9]+$ && "$NODE_LTS_VERSION" -lt 20 ]]; then
```

Comment:
nit: 

According to the [new npm release](https://www.npmjs.com/package/npm), only node versions `20.5.0` and above are supported by latest. I doubt we will use  a node version between `20` and `20.5.0` for any of our tests, but would we want to do a more involved check here?

Reply:
We typically don't specify version other than the major so we're always automatically ingesting the latest of each. The exception is the performance tests, which pin an exact version but those will remain on 18.16 for a long while and already have to pin to npm 9 because npm 10 only supported 18.17+, so at that time we will repeat the pattern for whatever the matching npm version is

Reply:
I see, so we won't be running any tests in between node versions 20 and 20.5.0.